src/components/Sidebar/Sidebar.jsx

```diff
@@ -9,8 +9,24 @@ import { SidebarFooter } from "./Sidebar.Footer";
 import { SidebarLink } from "./Sidebar.Link";
 import { SidebarMenu } from "./Sidebar.Menu";
 import { SidebarHeading } from "./Sidebar.Heading";
+import { Drawer } from "../Drawer";
+import { BellIcon } from "../../icons/BellIcon";
+import { AnnounceIcon } from "../../icons/AnnounceIcon";
+
+export const Sidebar = ({
+    children,
+    className,
+    footer,
+    notifications,
+    announcements,
+    notificationsContent,
+    announcementsContent,
```

Comment:
@nemanjakrstic These are a lot of extra props. Do you have a better solution to make this cleaner?

Reply:
Maps to src/pages/MenuSidebar.tsx

Reply:
@rushi, yes, components: https://github.com/xola/ui-kit/blob/master/src/components/DatePicker/DatePicker.jsx#L33

Reply:
@sachinp9797 Also, "announcements" feel like business logic thing. Make it more abstract. This is a UI kit